### DISCUSSION TITLE
Problems in launching SU2

### DISCUSSION THREAD

**Discussion by MarcoSalvat** [su2code/SU2#1580] [Q&A]
Hi everyone, when I try to run the SU2_CFD command I get this error as an output:

> Ignoring PCI device with non-16bit domain.
Pass --enable-32bits-pci-domain to configure to support such devices
(warning: it would break the library ABI, don't enable unless really needed).
Ignoring PCI device with non-16bit domain.
Pass --enable-32bits-pci-domain to configure to support such devices
(warning: it would break the library ABI, don't enable unless really needed).
[marco-gs66-stealth-11ue:13479] *** An error occurred in MPI_Win_create
[marco-gs66-stealth-11ue:13479] *** reported by process [3897098241,0]
[marco-gs66-stealth-11ue:13479] *** on communicator MPI_COMM_WORLD
[marco-gs66-stealth-11ue:13479] *** MPI_ERR_WIN: invalid window
[marco-gs66-stealth-11ue:13479] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[marco-gs66-stealth-11ue:13479] ***    and potentially your MPI job)

**To reproduce**
Firstly, I configure su2 by `python3 ./meson.py build -Denable-autodiff=true -Dwith-mpi=enabled --prefix=/home/ubuntu/CFD/SU2`.

![Schermata_1](https://user-images.githubusercontent.com/95415784/160251640-efbb1e1b-d330-4480-9f38-c8ac2e0734d2.png)
![Schermata_2](https://user-images.githubusercontent.com/95415784/160251641-1c6e1f17-04a9-4c3c-84b3-ef0b400b26dc.png)

Then i use  `./ninja -C build install` to compile it, and some warning messages appear:

![Schermata_3](https://user-images.githubusercontent.com/95415784/160251767-95b81ae5-88f3-4d79-90a8-9feadebe5edc.png)

When the compilation is finished, I try to run `SU2-CFD` to do a test and I get the error.

![Schermata_4](https://user-images.githubusercontent.com/95415784/160251835-f7a58b5a-c5fa-4193-819b-f7319816f50b.png)

**Comment by TobiKattmann** [comment]
The `SU2_CFD` binary requires a configuration (.cfg) to be run with. In the SU2 code root please go into the folder `QuickStart` and run `/home/ubuntu/CFD/SU2/bin/SU2_CFD inv_NACA0012.cfg` if that runs and produces meaningful output you're good to go. The installation is successful for you so I am somewhat confident this will work

Concerning compiler warnings: In AD builds (autodiff=true) and for the external packages (like cgns) the warnings are not an issue and very normal

On another note `python3 ./meson.py ...` is not necessary you would either do `python3 meson.py ...` or `./meson.py`. If you look at the first line of `meson.py` you'll see `!/usr/bin/env python3` which already specifies for the `./` command which executable to call the file with ... in the present case `./meson.py` would be equivalent with `/usr/bin/env python3 meson.py`. That concept is called `shebang` if you want to find resource on that

**Reply by TobiKattmann** [reply]
And Hi Marco of course :)
